lib/elixir/lib/kernel.ex

```diff
@@ -2544,6 +2554,11 @@ defmodule Kernel do
 
       get_and_update_in(opts, [:foo, :bar], &{&1, &1 + 1})
 
+  This also works with nested structs and the `struct.path.to.value` way to specify
+  paths"
```

Comment:
```suggestion
  paths:
```

Reply:
argh what a stupid mistake, thank you :green_heart: